pnpm-workspace.yaml

```
@@ -1,3 +1,4 @@
 packages:
   - 'packages/*'
   - 'tools-public'
+  - 'contributor-dashboard-legacy'
```

We could likely adopt the same folder structure as in https://github.com/mui/mui-private https://github.com/mui/mui-private/blob/6fd8a3a565e1ae51a7383fe1a85506bd36dc55f9/pnpm-workspace.yaml#L2

```suggestion
  - 'apps/*'
```